### ISSUE TITLE
Ones are not always one

### ISSUE THREAD

**Issue by omegahm** [bh107/bohrium#79]
With this:

``` python
import numpy as np
print np.ones((2, 1), dtype=np.float64) * 1.0
```

Running without Bohrium (`python ones.py`) yields:

``` python
[[ 1.]
 [ 1.]]
```

Running with Bohrium (`python -m bohrium ones.py`) yields:

``` python
[[  1.97626258e-323]
 [  1.97626258e-323]]
```

If I change `numpy` to `bohrium` it gives me **another wrong answer**:

``` python
[[  6.95321425e-310]
 [  6.95321425e-310]]
```

---

I'm running Mac OSX 10.11.4 with Python 2.7.11 and NumPy 1.10.4.

**Comment by omegahm** [comment]
Hmm... it seems to have something to do with the shape `(2, 1)`. 

If I do:

```python
import bohrium as np
print (np.ones(2, dtype=np.float64) * 1.0)[..., np.newaxis]
```

I get the correct result.

**Comment by omegahm** [comment]
This is linked to https://github.com/bh107/benchpress/issues/5, where we have this kind of calculation.

**Comment by madsbk** [comment]
This problem can be triggered in linux by the following code:
```python
import bohrium as np
a = np.ones((2, 1), dtype=np.float64)
a.strides = (8,0)
a = a * 1
print a
```

The  problem relates to the CPU-VE and the use of arrays with 0-stride'ed dimensions.

**Comment by omegahm** [comment]
If I turn off `jit_contraction` (ie. set `BH_CPU_JIT_LEVEL=2`) it gives the correct results.

**Comment by madsbk** [comment]
Fixed in #140
